lib/rpmte.cc

```diff
@@ -93,7 +93,7 @@ void rpmteCleanDS(rpmte te)
 {
     te->thisds = rpmdsFree(te->thisds);
     te->provides = rpmdsFree(te->provides);
-    te->requires = rpmdsFree(te->requires);
+    te->requires_ = rpmdsFree(te->requires_);
```

Comment:
This is already merged, but I would have probably gone with changing all of these to `dep_<deptype>` to make it consistent and not look stupid.

Reply:
It's an eyesore alright, but we'll really want to move all these rpmte's to a container of some sort anyhow so that'll go away sooner than later.